### PR TITLE
#issue-1074 Added image selecting/highlighting when clicking on the grey area or license label

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
### Description (*)

This PR adds Media Gallery image item selecting/highlighting styles when clicking or hovering grey area or a license label.

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1074: The image is not selected/highlighted when clicking on it's grey area

### Manual testing scenarios (*)

1. From Admin go to Content - Pages, click Add New Page
2. Expand Content, click Show / Hide Editor, click Insert Image...
3. Try to select an image by clicking on its grey area

### CODE REVIEW

File: MediaGalleryUi/view/adminhtml/web/template/grid/columns/image.html [hunk 1]
```diff
@@ -6,10 +6,12 @@
 -->
 <div class="media-gallery-wrap" collapsible>
     <div class="media-gallery-image">
-        <div class="masonry-image-block media-gallery-image-block" attr="'data-id': $col.getId($row())" css="{ selected: isSelected($row()) }">
+        <div class="masonry-image-block media-gallery-image-block"
+             attr="'data-id': $col.getId($row())"
+             css="{ selected: isSelected($row()) }"
+             click="function(){ if (!$collapsible.opened()) {select($row())} }" >
```
Comment (VladimirZaets) [inline MediaGalleryUi/view/adminhtml/web/template/grid/columns/image.html]: Can you please move this function from view to model method? It's not good to have some logic in html template. thanks

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @serhiyzhovnir. Thanks for collaboration. Please take a look at my comment below

**Comment by sivaschenko** [comment]
Also, I think that we should not select images on hover, only on click

**Comment by serhiyzhovnir** [comment]
Hi @VladimirZaets @sivaschenko 
The requested changes were added.
Please, check this PR again.
Thank you!

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**